lib/controllers/foods-controller.js

```diff
@@ -0,0 +1,83 @@
+const Food = require('../models/food')
+
+function show (request, response) {
+  Food.find(request.params.id)
+  .then((data) => {
+    let food = data.rows[0]
+
+    if(food == null){
+      response.sendStatus(404)
+    } else {
+      response.json(food)
+    }
+  });
+};
+
+function index (request, response) {
+  Food.all()
+  .then((data) => {
+    let food = data.rows
+
+    if(food == null){
+      response.sendStatus(404)
+    } else {
+      response.json(food)
+    }
+  });
+};
+
+
+function create (request, response) {
+  let name = request.body.food_name
+  let calories = request.body.calories
+
+  if(name == null) {
```

Comment:
i think you can do the same here as you do with calories on line 35. `if(!name)`